### ISSUE TITLE
Music playback pauses while playing

### ISSUE THREAD

**Issue by BatteryPine** [DeaDBeeF-Player/deadbeef#2747]
### Steps to reproduce the problem
Play a playlist of music, stop half-way through and then return to the playlist after some minutes have passed.
### What's going on? Describe the problem in as much detail as possible.
Sometimes when I stop playing a track in a playlist and then continue back where I was in the playlist, the song will play for a few seconds and then buffer for a few seconds before resuming playback. I've noticed this happen with FLAC files and it went away the first time by compressing the first album I had this issue with to level 8, but the second time it still happened even after compressing the files again and I could reproduce it. I haven't tested it out with lossy files (MP3, Vorbis, Opus, AAC), so I have no clue if this only affects the playback of lossless files or not.
### Information about the software:

Deadbeef version: 1.8.8
OS: Kubuntu 21.10

**Comment by Alexey-Yakovenko** [comment]
Are you running pipewire sound server?

**Comment by BatteryPine** [comment]
No, I'm using the ALSA output plugin and I have my output device set to Playback/recording through the PulseAudio sound server.
![Screenshot_20220216_154227](https://user-images.githubusercontent.com/22846912/154353910-d7e4543d-a782-454a-a4ef-804861a32703.png)

**Comment by Alexey-Yakovenko** [comment]
According to google search, pipewire is used by default in 21.10, and this seems to be the problem.

Deadbeef doesn't support pipewire out of the box, and pipewire itself breaks ALSA support in deadbeef.

You have two options:
* Use pulseaudio in Deadbeef instead of ALSA output -- Pulseaudio emulation is reported to work better with pipewire, than ALSA
* Install this pipewire plugin for deadbeef: https://github.com/saivert/ddb_output_pw

**Comment by lordoff** [comment]
> No, I'm using the ALSA output plugin and I have my output device set to Playback/recording through the PulseAudio sound server. ![Screenshot_20220216_154227](https://user-images.githubusercontent.com/22846912/154353910-d7e4543d-a782-454a-a4ef-804861a32703.png)
You don't need any pulse or pipewire server if you choose ALSA output plugin. 
Just specify in tab "Output device" your! device... USB DAC for example, if you have.
![Screenshot_2022-02-16_23-53-37](https://user-images.githubusercontent.com/23076154/154364220-dd36e92e-fe74-46f3-8a6b-3333bf152950.png)
